build/images/codegen/Dockerfile

```diff
@@ -22,6 +22,8 @@ ARG K8S_VERSION=1.17.6
 ARG KUBEOPENAPI_VERSION=v0.0.0-20200410145947-bcb3869e6f29
 
 RUN go get k8s.io/code-generator/cmd/client-gen@kubernetes-$K8S_VERSION && \
+    go get k8s.io/code-generator/cmd/lister-gen@kubernetes-$K8S_VERSION && \
+    go get k8s.io/code-generator/cmd/informer-gen@kubernetes-$K8S_VERSION && \
```

Comment:
these lines already exist below; merge conflict that wasn't resolved properly?

Reply:
Yes, it was not considered a conflict since added at different spots. Removed those lines. Thanks for spotting this.